src/native/libs/System.Native/pal_threading.c

```diff
@@ -310,23 +311,6 @@ uint64_t SystemNative_GetUInt64OSThreadId(void)
 // Returns (uint32_t)-1 when the implementation does not know how to get the OS thread ID.
 uint32_t SystemNative_TryGetUInt32OSThreadId(void)
```

Comment:
Delete the includes a few lines above. Also, change `SystemNative_GetUInt64OSThreadId` above to call into minimal